README.md

```diff
@@ -57,9 +65,22 @@ With the CodePush plugin installed, configure your app to use it via the followi
     <meta http-equiv="Content-Security-Policy" content="default-src https://codepush.azurewebsites.net 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *" />
     ```
    
+4. Finally, double-check that you already have the [`cordova-plugin-whitelist`](https://github.com/apache/cordova-plugin-whitelist) plugin installed (most apps will). To check this, simply run the following command:
+
+    ```shell
+    cordova plugin
```

Comment:
`cordova plugin list` is the actual command, although the `cordova plugin` command defaults to that as well.